BO4E/BO/Auftrag.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Text.Json.Serialization;
+using BO4E.COM;
+using BO4E.ENUM;
+using BO4E.meta;
+using Newtonsoft.Json;
+using ProtoBuf;
+
+namespace BO4E.BO
+{
+    /// <summary>
+    /// Ein Auftrag beschreibt einen Vorgang, der von einem anderen Marktpartner auszuführen ist. 
+    /// </summary>
+    [NonOfficial(NonOfficialCategory.REGULATORY_REQUIREMENTS)]
+    [ProtoContract]
+    public abstract class Auftrag : BusinessObject
+    {
+        /// <summary>
+        /// workaround
+        /// </summary>
+        [System.Text.Json.Serialization.JsonIgnore]
+        [Newtonsoft.Json.JsonIgnore]
+        [ProtoMember(4, Name = nameof(Ausfuehrungsdatum))]
+        [CompatibilityLevel(CompatibilityLevel.Level240)]
+        protected DateTime _Ausfuehrungsdatum
+        {
+            get => Ausfuehrungsdatum.UtcDateTime;
+            set => Ausfuehrungsdatum = DateTime.SpecifyKind(value, DateTimeKind.Utc);
+        }
+
+        /// <summary>
+        /// Das Ausführungsdatum beschreibt zu welchem Zeitpunkt ein Auftrag ausgeführt werden soll.
+        /// </summary>
+        [JsonProperty("ausfuehrungsdatum", Required = Required.Always)]
+        [JsonPropertyName("ausfuehrungsdatum")]
+        [ProtoIgnore]
+        public DateTimeOffset Ausfuehrungsdatum { get; set; }
```

Comment:
Hier könnten wir im Zuge von .net6 auch überlegen, ob wir dann auf DateOnly gehen

Reply:
würde ich aber in einem separaten PR machen (also .net6)